### ISSUE TITLE
Provide Option for Looping Setting

### ISSUE THREAD

**Issue by michael-hawker** [windows-toolkit/Lottie-Windows#53]
It'd be nice if there was a toggle button next to the time slider to control if the animation looped (this could modify the generated code as well).

**Comment by sohchatt** [comment]
Since the LottieViewer, for now, is primarily used to test the visual correctness of an animation rendered on Windows, we decided to keep the UI simple and limit playback options. We opted to loop by default since some json files may have a very short duration (for instance, icon animation are typically < 1 sec) which makes it hard to understand what's happening in one go. If this feature is valuable to users, we'd be happy to add this to our backlog for the LottieViewer app.

Also, the generated code has no notion of animation playback and is solely a C#/ C++ representation of the AfterEffects composition. All playback properties, including IsLoooping, are controlled via the AnimaterVisualPlayer XAML control that hosts the LottieVisualSource or codegen-ed class.
